releasenotes/releasenotes_v4.rst

```diff
@@ -256,6 +256,16 @@ Database
       If you have questions about the migration, please :ref:`contact us<vonk-contact>`.
 
 
+   The required migrations for SQL Server will be applied automatically if ``AutoUpdateDatabase=true`` in the settings. Otherwise, or if automatic migration times out, you can run them manually. The scripts are located in the directory ``./sqlserver``. You can see the list of applied migrations in table ``[vonk].[schemainfo]``. The upgrade requires the following migrations:
```

Comment:
I would link to https://docs.fire.ly/projects/Firely-Server/en/latest/maintenance/upgrade.html#migrations, from the word 'manually.